SortingAlgorithms.java

```diff
@@ -0,0 +1,7 @@
+public interface SortingAlgorithms {
+    String[] bubbleSort(String[] arr);
+
+    String[] selectionSort(String[] arr);
+

```

Comment:
Error in this line

Reply:
There was no error. Already checked it. Go buy some glasses